src/app/modules/item/components/item-children-edit/item-children-edit.component.html

```diff
@@ -42,13 +42,16 @@
           </div>
         </ng-template>
       </p-table>
+
+      <span *ngIf="!data.length">No children for this chapter.</span>
+
+      <alg-add-content [allowSkills]="this.itemData?.item?.type === 'Skill'"></alg-add-content>
```

Comment:
* Don't use "this" in templates. 
* Item is not optional

Reply:
![image](https://user-images.githubusercontent.com/22669257/100365222-7c174d80-2fff-11eb-9685-3ba61a59d2ca.png)
I really understand why but it won't let me compile without that '?'

Reply:
My bad, I forgot that '?'  doesn't work in the same way it does in a ts file. Can I use '!' here?

Reply:
Using a '!' can cause problem if this component is used incorrectly so I've used another way